### PR TITLE
Update proposal to allow individual stopping of clustering and reconciliation

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/moco/issues/564

### CODE REVIEW

File: docs/designdoc/clustering_stop.md [hunk 1]
```diff
@@ -71,27 +74,69 @@ status:
...
+Also, the `ClusterManager` does not stop even if the reconcile process is stopped.
+
+The `ReconciliationStopped` gets added to `.status.conditions` of MySQLCluster.
+
+```yaml
+status:
+  conditions:
+    - type: ReconciliationStopped
+      status: "True"
+      lastTransitionTime: 2018-01-01T00:00:00Z
```
Comment (masa213f) [inline docs/designdoc/clustering_stop.md:116]: When a MySQL cluster is normal, I want all ".status.conditions" to be "true".
Could you rename the type of the condition to reverse the boolean value?

Also, please rename the following names.

- Header of `kubectl get mysqlcluster`
- Annotation key
- Metrics name

**Review by masa213f — CHANGES_REQUESTED** [review]

**Review by masa213f — DISMISSED** [review]
LGTM

File: docs/designdoc/clustering_stop.md [hunk 2]
```diff
@@ -71,27 +74,70 @@ status:
...
 ```
 
-Also, `STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
+Also, `CLUSTERING STOPPED` will be added to the table displayed when you do `kubectl get mysqlcluster`.
 
 ```console
 $ kubectl get mysqlcluster
-NAME   AVAILABLE   HEALTHY   STOPPED   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   LAST BACKUP
-test   Unknown     Unknown   True      0         2                                   <no value>
+NAME   AVAILABLE   HEALTHY   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   CLUSTERING STOPPED   RECONCILE STOPPED   LAST BACKUP
```
Comment (zoetrope) [inline docs/designdoc/clustering_stop.md:81]: Is this correct?
The following section uses "CLUSTERING ACTIVE" and "RECONCILE ACTIVE".

Comment (d-kuro) [reply docs/designdoc/clustering_stop.md]: Thanks 🙏 
I fixed: https://github.com/cybozu-go/moco/pull/576/commits/06e8204b96fddc01398fcca7bd3cabe613cdb902

**Review by zoetrope — APPROVED** [review]
LGTM

**Pull request merged**